var/spack/repos/builtin/packages/sst-elements/package.py

```diff
@@ -56,7 +57,7 @@ class SstElements(AutotoolsPackage):
     variant("otf", default=False, description="Build with OTF")
     variant("otf2", default=False, description="Build with OTF2")
 
-    depends_on("python", type=("build", "run"))
+    depends_on("python@:3.12", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("python@:3.11", type=("build", "run"))
```
@berquist You committed this with: "SST does not work with Python 3.12 yet"

If I'm not totally mistaken, you'd have to update this commit to use `python@:3.11` instead.